modules/webgl/src/classes/resource.js

```diff
@@ -295,15 +295,25 @@ export default class Resource {
   }
 
   _trackAllocatedMemory(bytes, name = this[Symbol.toStringTag]) {
-    const stats = lumaStats.get('Memory Usage');
+    this._trackAllocatedMemoryForContext(bytes, name);
```

Comment:
I think it will be confusing to have three sets of stats in the same stats object,,. Maybe drop this line?

Reply:
Well... the reason for that `_getMetrics()` function even existing is that we do not guarantee stable names in the stats object, it is just a "bag" of data. So deck.gl maps those "loose" fields to a well-documented, typed object.

Which means that function implementation is the place to change if the names in the stats object change.

A possible alternative is to have multiple stats object and just add a reference to each stats object to its gl context or canvas.

Then you can still support the global object that counts resources from all contexts.